memegen/templates/overview.html

```diff
@@ -65,7 +65,7 @@
       {% for img in imgs %}
         <label class="meme-item">
           <a href="{{ img.url }}"><img src="{{ img.url }}" class="meme-img"/></a>
-          <a href="{{ img.link }}">{{ img.link }}</a>
+          <a href="{{ img.url }}">{{ img.url }}</a>
```

Comment:
This is actually altering the existing behavior. The purpose for these two links:

- click on the image to get the image URL
- click on the link to get the API page of links